pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -356,21 +377,25 @@ func (e *cloudWatchExecutor) executeLogAlertQuery(ctx context.Context, req *back
 	resp := backend.NewQueryDataResponse()
 
 	for _, q := range req.Queries {
-		model, err := simplejson.NewJson(q.JSON)
+		var model LogQueryJson
+		err := json.Unmarshal(q.JSON, &model)
 		if err != nil {
 			continue
 		}
 
-		model.Set("subtype", "StartQuery")
-		model.Set("queryString", model.Get("expression").MustString(""))
+		model.Subtype = "StartQuery"
+		model.QueryString = model.Expression
 
-		region := model.Get("region").MustString(defaultRegion)
-		if region == defaultRegion {
+		region := defaultRegion
+
+		if model.Region != "" {
```

Comment:
I think the region in the model can be `default`, so this should be `model.Region != "" && model.Region != defaultRegion`.

Also this can be refactored to not have the `else` by doing:
```
region := model.Region
if region == "" || region == defaultRegion {
...
```